core/src/main/resources/Layouts.yml

```diff
@@ -12,6 +11,13 @@
 # For warns you have also the variable %COUNT% which will be
 # replaced with the current amount of warns the player already received
 # Example usage: /ban Leoko @ExampleLayout
+
+# Use Message Layouts (If disabled, this will disable Tab-completion of Message Layouts aswell. Any entries within the Message section will be ignored)
```

Comment:
```suggestion
# Use Message Layouts (If disabled, this will disable Tab-completion of Message Layouts as well. Any entries within the Message section will be ignored)
```